pkg/imageupdater/airflow.go

```diff
@@ -2,23 +2,42 @@ package imageupdater
 
 import (
 	"context"
+	"database/sql"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/nais/knorten/pkg/database/gensql"
 )
 
 const (
-	airflowImageNameKey = "images.airflow.repository"
-	airflowImageTagKey  = "images.airflow.tag"
+	airflowBaseImageNameKey = "images.airflow.repository"
+	airflowBaseImageTagKey  = "images.airflow.tag"
+	airflowSyncImageNameKey = "images.gitSync.repository"
+	airflowSyncImageTagKey  = "images.gitSync.tag"
+	airflowGlobalEnvKey     = "env"
```

Comment:
```suggestion
	imagesAirflowRepositoryKey = "images.airflow.repository"
	imagesAirflowTagKey  = "images.airflow.tag"
	imagesGitSyncRepositoryKey = "images.gitSync.repository"
	imagesGitSyncTagKey  = "images.gitSync.tag"
	envKey     = "env"
```

Er det bedre at disse variablene matcher Helm-nøkkelen i chartet? Vi er jo i fila Airflow, men kanskje dumt siden man er i en annen pakke.

Reply:
```suggestion
	airflowBaseImagesRepositoryKey = "images.airflow.repository"
	airflowBaseImagesTagKey  = "images.airflow.tag"
	airflowGitSyncImagesRepositoryKey = "images.gitSync.repository"
	airflowGitSyncImagesTagKey  = "images.gitSync.tag"
	airflowEnvKey     = "env"
```